packages/runtime/test/src/interpreter/opcode-list.ts

```diff
@@ -5700,7 +5716,16 @@ describe("Teal Opcodes", function () {
 			const op = new ExtractUint16([], 1);
 			op.execute(stack);
 
-			const expected = bigEndianBytesToBigInt(new Uint8Array([3, 4]));
+			let expected = bigEndianBytesToBigInt(new Uint8Array([3, 4]));
+			assert.equal(stack.pop(), expected);
+
+			// long byte
+			stack.push(longByte);
+			stack.push(300n);
+
```

Comment:
There is a lot of empty lines I see. You should remove them wherever not needed.

Reply:
Done